disnake/flags.py

```diff
@@ -1633,3 +1728,72 @@ class ChannelFlags(BaseFlags):
     def pinned(self):
         """:class:`bool`: Returns ``True`` if the thread is pinned."""
         return 1 << 1
+
+
+@fill_with_flags()
+class AutoModKeywordPresets(ListBaseFlags):
+    """
+    Wraps up the pre-defined auto moderation keyword lists, provided by Discord.
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two flags are equal.
+        .. describe:: x != y
+
+            Checks if two flags are not equal.
+        .. describe:: hash(x)
+
+            Return the flag's hash.
+        .. describe:: iter(x)
+
+            Returns an iterator of ``(name, value)`` pairs. This allows it
+            to be, for example, constructed as a dict or a list of pairs.
+            Note that aliases are not shown.
```

Comment:
missing operators added in #605

Reply:
Those might need to be tested/re-implemented on ListBaseFlags in this pull.

Reply:
Updated documentation in 3e406b13b20fbfc8ec83d4ea858b8d2f408f2745, no implementation changes needed as it uses `BaseFlags.value` internally.